### PR TITLE
Aftershock vehicle cleanup and Blaze Industries vehicle migration.

### LINKED ISSUE #49384: Aftershock Vehicles not tiling correctlty.
# Describe the bug
The vehicles from aftershock are not tiling correctly. It seems the parts are starting as the non-aligned generic varieties. This makes the vehicles look pretty bad, and required a teardown and rebuild of the affected parts to look right. 

# Steps To Reproduce
1.Start a new world with aftershock enabled.
2.Turn on a tile set.
2. Spawn in or find any of the atomic cars or even the electric semi.
3. Notice the vehicle is tiled wrong.

# Expected behavior
The vehicle spawns in with the right parts for tiling.

# Screenshots
badvehicletile

# Versions and configuration
- OS: Linux
    - OS Version: LSB Version: n/a; Distributor ID: ManjaroLinux; Description: Manjaro Linux; Release: 21.0.7; Codename: Ornara; 
- Game Version: fc3689c [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions],
    Aftershock [aftershock]
]

### PR DESCRIPTION
**Pull request by xerenogan**
#### Summary
Mods "Aftershock vehicle cleanup and Blaze Industries vehicle migration."

#### Purpose of change
Fixes #49384
I noticed the aftershock vehicles tiled very poorly, and when looking to fix that, decided to clean up the json for the aftershock vehicles. I was also asked to migrate some of the vehicles to blaze industries, which this accomplishes.

#### Describe the solution
The solution was implemented in 3 stages.
  1.Going through the vehicle definitions in aftershock and redid them to be a little neater, and take up fewer lines.
  2.Rewriting the definitions of the vehicles to use more appropriate parts that tile as intended.
  3.Migrating the Sport Bike, Grenade Launcher APC, and Electric Semi vehicles to Blaze Industries along with their requisite dependencies.

#### Describe alternatives you've considered
None
#### Testing
Copying the affected files over to a working copy and running four worlds, one with aftershock, one with blaze industries, one with both and one with neither. Observed lack of error messages on startup and spawned in the relevant vehicles and observed that they either did or did not show as an option when expected and that no error presented itself when spawned in.
This procedure was followed at each step as well as when finished. 

#### Additional context
Before.
bustvehicles

After.
bustvehicles2

### CODE REVIEW

File: data/mods/BlazeIndustries/vehicles/vehicles.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      { "x": -3, "y": 0, "parts": [ "hdframe_horizontal", "hddoor_trunk", "muffler" ] },
+      { "x": -3, "y": 0, "part": "plating_military" },
+      { "x": -3, "y": -1, "parts": [ "hdframe_sw", "hdboard_sw", "plating_military" ] },
+      { "x": -3, "y": -1, "parts": [ "wheel_mount_heavy", "wheel_armor" ] }
+    ],
+    "items": [
+      { "x": 0, "y": 0, "chance": 5, "items": [ "id_military" ] },
+      { "x": -2, "y": 0, "chance": 5, "item_groups": [ "fuel_diesel" ] }
+    ]
+  }
```
Comment (Maleclypse) [inline data/mods/BlazeIndustries/vehicles/vehicles.json:185]: I'm not seeing the grenade launcher APC in this file, only the humvee?

Comment (Maleclypse) [reply data/mods/BlazeIndustries/vehicles/vehicles.json:185]: Nevermind I think that vehicle is already gone.

Comment (xerenogan) [reply data/mods/BlazeIndustries/vehicles/vehicles.json:185]: I didn't know there was a grenade launcher apc in aftershock. Is it vanilla?

Comment (xerenogan) [reply data/mods/BlazeIndustries/vehicles/vehicles.json:185]: Oh, I think I see the one you mean. Is "aapc-gl" the one you meant?

**Review by Maleclypse — APPROVED** [review]

**Review by Maleclypse — CHANGES_REQUESTED** [review]
data/mods/Aftershock/items/gun/grenade.json → ...ds/BlazeIndustries/items/gun/grenade.json

Can you move a copy of the contents of this file into Aftershock's Obsolete file. Otherwise it will destroy save games.

**Comment by xerenogan** [comment]
There we go, copied to aftershocks obsolete item file.
Sorry it took me so long to respond, I just got home from work.

**Review by Maleclypse — APPROVED** [review]

**Comment by Maleclypse** [comment]
> There we go, copied to aftershocks obsolete item file.
> Sorry it took me so long to respond, I just got home from work.

Thanks and no stress on real life coming first! :)

**Pull request merged**